src/components/network/network-table.js

```diff
@@ -69,21 +80,28 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
+const ROW_HEIGHT = 48;
+
 const NetworkTable = (props) => {
     const classes = useStyles();
 
+    const notifiedDisplayedColumns = useSelector(
+        (state) => state.displayedColumns
+    );
+
     const [tabIndex, setTabIndex] = useState(0);
     const [lineEdit, setLineEdit] = useState({});
     const [rowFilter, setRowFilter] = useState(undefined);
     const [popupSelectColumnNames, setPopupSelectColumnNames] = useState(false);
+    const [selectedColumnsNames, setSelectedColumnsNames] = useState(null);
+    const [
+        backupSelectedColumnsNames,
+        setBackupSelectedColumnsNames,
+    ] = useState(null); // Optimization to reduce database access
```

Comment:
it's better to use a ref if the component doesn't need to be rendered when the value changes

Reply:
yes